Cargo.toml

```diff
@@ -7,7 +7,9 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-serde_json = "1"
+serde_json = "1.0.91"
+clap = { version = "4.0.32", features = ["derive"] }
```

Comment:
4.1.1 is out - https://github.com/clap-rs/clap/blob/master/CHANGELOG.md#411---2023-01-14